tests/virt_autotest/proxymode_conn_slave.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: proxymode_conn_slave: Connect physical machine thru proxy machine.
+# Maintainer: John <xgwang@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base "proxymode";
+
+sub run() {
+    my $self         = shift;
+    my $ipmi_machine = get_var("IPMI_HOSTNAME");
+    die "There is no ipmi ip address defined variable IPMI_HOSTNAME" unless $ipmi_machine;
+    $self->connect_slave($ipmi_machine);
+    $self->restart_host($ipmi_machine);
+    assert_screen "proxy_virttest-pxe", 600;
+    sleep 3;
```

Comment:
I want to suspend 3 seconds after last operations, and make sure the PXE is available status and can receive keyboard from user.